docs/conf/prague/2020/news/cfp-open.rst

```diff
@@ -0,0 +1,46 @@
+:template: {{year}}/generic.html
+
+.. post:: June 22, 2020
+   :tags: prague-2020, website, cfp, tickets
+
+Announcing Prague Call for Proposals
+====================================
+
+Today we are announcing our `Call for Proposals <https://www.writethedocs.org/conf/prague/2019/cfp/>`_.
+
+Due to the circumstances that you're all familiar with, we're opening it later than usual, and for a shorter length of time.
```

Comment:
This feels a bit glib -- we should likely mention explicitly moving it online, at least.